packages/bridge-ui/src/erc20/bridge.spec.ts

```diff
@@ -195,9 +195,12 @@ describe("bridge tests", () => {
       opts.tokenAddress,
       opts.amountInWei,
       BigNumber.from(100000),
-      opts.processingFeeInWei,
+      0,
       "0xfake",
       opts.memo
+      // {
+      //   value: opts.processingFeeInWei,
```

Comment:
same here

Reply:
yes its commentedo ut til protocol is fixed